BepressImportPlugin.inc.php

```diff
@@ -85,6 +92,10 @@ function executeCLI($scriptName, &$args) {
 			echo __('plugins.importexport.bepress.unknownUser', array('username' => $editorUsername)) . "\n";
 			exit();
 		}
+		// FIXME: This attaches the associated user to the request and is a workaround for no users being present
+		//     when running CLI tools. This assumes that given the username supplied should be used as the
+		//  authenticated user. To revisit later.
+		Registry::set('user', $editor);
```

Comment:
I had to add this workaround and a `setContext()` to the `NativeImportExportPlugin`, perhaps we should move this into the `ImportExportPlugin` later 🤔